recipe/meta.yaml

```
@@ -12,32 +12,36 @@ source:
 
 build:
   number: 0
-  script: python -m pip install --no-deps --ignore-installed .
+  script: python -m pip install --no-deps --no-build-isolation --ignore-installed .
 
 requirements:
   build:
     - {{ compiler('c') }}
   host:
     - python
     - pip
-    - numpy
+    - numpy {{ numpy }}
     - setuptools
     - wheel
   run:
     - python
-    - {{ pin_compatible('numpy') }}
+    - numpy >=1.6,<3
     - scipy >=0.9
 
 test:
   requires:
     - pip
+    - numpy {{ numpy }}
```

```suggestion
    - numpy
```
A plain `numpy` will be enough